src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -516,7 +516,7 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_arccosh)
 
 // arctanh
 MXNET_OPERATOR_REGISTER_UNARY(arctanh)
-.describe(R"code(Inverse hyperbolic tangent, element-wise.
+.describe(R"code(Returns element-wise inverse hyperbolic tangent value of the input.
```

Comment:
"Returns the element-wise inverse hyperbolic tngent of the input array, computed element-wise."